internal/cluster/workflow/k8s_health_check_activity.go

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2020 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workflow
+
+import (
+	"context"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+const K8sHealthCheckActivityName = "k8s-health-check"
+
+type K8sHealthCheckActivityInput struct {
```

Comment:
I think we don't need the `k8s` prefix here either. BUT: I'd keep it in the activity name.